build-support/githooks/prepare-commit-msg

```diff
@@ -24,8 +24,10 @@ NUM_RUST_FILES=$(echo "${CHANGED_FILES})" | grep -c -E \
   -e "^build-support/bin/generate_github_workflows.py")
 
 NUM_RELEASE_FILES=$(echo "${CHANGED_FILES})" | grep -c -E \
+  -e "^pants.toml" \
```

Comment:
The two edits here are because edits to either of these files do end up impacting validation in one way or another.